### ISSUE TITLE
Email start date

### ISSUE THREAD

**Issue by madu2003** [jstedfast/MailKit#375]
Hi ,

How do you read an email account (Imap) from a particular time forward? I am using the following code. But this reads all emails on a particular day. I need to update the time read so that I do not have to read any previous emails from a time on a single day.

DateTime ReadSDate = System.DateTime.Today;
var query = SearchQuery.DeliveredAfter(ReadSDate);
foreach (var uid in inbox.Search(query))

**Comment by jstedfast** [comment]
The IMAP protocol does not support searching on a particular time of the day, it only supports searching based on a day.

If you want to read only messages delivered on a day after a certain specified time, you'll need to do the search you are currently doing and then further filter the results down by doing manual date comparisons.

for example:

```csharp
var results = inbox.Search (SearchQuery.DeliveredAfter (DateTime.Today));
var uids = new UniqueIdSet ();

foreach (var info in inbox.Fetch (results, MessageSummaryItems.InternalDate | MessageSummaryItems.UniqueId)) {
    if (info.InternalDate.Value >= searchDateTime)
        uids.Add (info.UniqueId);
}

// now `uids` is the set of messages that match what you are looking for...
```

**Comment by madu2003** [comment]
Thanks, Jeffrey. That worked nicely....! 

    On Friday, July 22, 2016 1:41 PM, Jeffrey Stedfast <notifications@github.com> wrote:
 

 Closed #375.—
